src/flat.rs

```diff
@@ -1443,37 +1443,6 @@ mod tests {
        assert_eq!(pixel_count, 100*100);
     }
 
-    #[test]
-    fn mutable_view() {
```

Comment:
That test had did some assertions other than constructing the mutable iterator. Can you replace

```
            #[allow(deprecated)]
            let pixel_count = view.pixels_mut()
                .enumerate()
                .map(|(idx, (_, _, pixel))| *pixel = LumaA([2*idx, 2*idx + 1]))
                .count();
```

with:

```
assert_eq!(view.dimensions(), (3, 3));
for idx in 0..9 {
    *view.get_pixel_mut(idx % 3, idx / 3) = LumaA([2*idx, 2*idx + 1]);
}
```